lib/src/resources/i_movie_repository.dart

```diff
@@ -0,0 +1,7 @@
+import '../models/movie.dart';
+
+abstract class IMovieRepository {
+  Future<MovieModel> fetchAllMovies();
```

Comment:
you change the name of the file movie_model to movie, but the class hasn't change, rename it to keep consistency

Reply:
done